src/gretel_trainer/runner.py

```diff
@@ -413,11 +414,6 @@ def train_partition(
 
         if "synthetics" in model_config["models"][0].keys():
 
-            model_config["models"][0]["synthetics"]["generate"] = {
-                "num_records": artifact.record_count,
-                "max_invalid": None,
-            }
```

Comment:
@zredlined Removing this restores the behavior of generating 5000 records during training, which is both [explicitly on the `synthetics/default` blueprint](https://github.com/gretelai/gretel-blueprints/blob/main/config_templates/gretel/synthetics/default.yml#L33) and is the default when unspecified, as in the case of CTGAN (both here in the trainer and in its blueprint)